src/Rings/mpoly.jl

```diff
@@ -192,6 +192,7 @@ function (S::Singular.Rationals)(a::fmpq)
   return S(b)
 end
 
+(Ox::MPolyRing)(f::MPolyElem) = convert(Ox, f)
```

Comment:
This confused me, until I realized that apparently in `Oscar`, we hijacked `convert` to use it for coercing / converting ring elements to a different parent. So we'd write `convert(R, x)` for what would be written as `R(x)` in AbstractAlgebra / Nemo / Singular -- or am I wrong?!  From that perspective, the above change is correct and desirable; but then I don't see why we keep the `convert` -- would perhaps be better to drop this in Oscar and change everything to the `R(x)` usage? Perhaps @fieker can explain?

Reply:
Seem this causes a test failure now (in the "extended" tests coming from the "experiments", active when using in Oscar dev mode since pR #214 was merged a bit earlier...
```
Got exception outside of a @test
  MethodError: Multivariate Polynomial Ring in x1, x2, x3 over Fraction field of Multivariate Polynomial Ring in t1, t2 over Rational Field(::fmpq_mpoly) is ambiguous. Candidates:
    (a::AbstractAlgebra.Generic.MPolyRing{T})(b::Union{RingElem, AbstractFloat, Integer, Rational}) where T<:Union{RingElem, AbstractFloat, Integer, Rational} in AbstractAlgebra.Generic at /home/runner/.julia/packages/AbstractAlgebra/aF5Iw/src/generic/MPoly.jl:4960
    (Ox::MPolyRing)(f::MPolyElem) in Oscar at /home/runner/work/Oscar.jl/Oscar.jl/src/Rings/mpoly.jl:195
  Possible fix, define
    (::AbstractAlgebra.Generic.MPolyRing{T})(::MPolyElem) where T<:Union{RingElem, AbstractFloat, Integer, Rational}
...
```

Reply:
I fixed this (and in the convert PR) and now I have an error in the Modstd tests.